src/render-dom.js

```diff
@@ -117,22 +117,23 @@ function makeEventsSelector(rootEl$, namespace) {
     return rootEl$
       .first()
       .flatMapLatest(rootEl => {
-        if (rootEl.length === 0) {
-          return Rx.Observable.empty()
-        }
         if (!namespace || namespace.length === 0) {
           return fromEvent(rootEl, eventName, useCapture)
         }
         const descendantSelector = namespace.join(` `)
-        if (matchesSelector(rootEl, descendantSelector)) { // is root Element
-          return fromEvent(rootEl, eventName, useCapture)
-        }
         const topSelector = namespace.join(``)
+        const roof = rootEl.parentElement
         return fromEvent(rootEl, eventName, useCapture).filter(ev => {
-          if (matchesSelector(ev.target, descendantSelector) ||
-              matchesSelector(ev.target, topSelector))
-          {
-            return isStrictlyInRootScope(ev.target)
+          for (let el = ev.target; el && el !== roof; el = el.parentElement) {
+            if (!isStrictlyInRootScope(el)) {
+              return false
```

Comment:
actually you can reuse the same use case i created for #80:
https://gist.github.com/laszlokorte/e2043a2544184d9aa928
it is not working anymore. Neither inner nor outer component get any of the events.